submissions/Eugene-CG/OOP-exercise/index.js

```diff
@@ -0,0 +1,144 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+   Code repository: https://github.com/Eugene-CG/a-tiny-JS-world
+   Web app: https://eugene-cg.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Inhabitant {
+  constructor(name, saying, gender) {
+    this.name = name;
+    this.saying = saying;
+    this.gender = gender;
+  }
+  getName() {
```

Comment:
Separating every other class property by different functions just to say the same thing is a bit of an overkill. Make it more simple. In this version, you can just use a string template to show some values. It is fine to break the final phrase for each Inhabitant in parts because different classes in the hierarchy will hold different info about the inhabitant, but do not do it like this.